co2eq/purchase/index.js

```diff
@@ -168,7 +176,7 @@ function conversionCPI(eurAmount, referenceYear, countryCodeISO2, datetime) {
  * @param {*} datetime - datetime of the activity
  */
 function extractCompatibleUnitAndAmount(lineItem, entry, countryCodeISO2, datetime) {
-  const isMonetaryItem = getAvailableCurrencies().includes(lineItem.unit);
+  const isMonetaryItem = Object.keys(exchangeRates2011.rates).includes(lineItem.unit);
```

Comment:
It seems to me that detecting that something is monetary is not the same as making sure a 2011 exchange rate is available.
Thus, I would revert this line. In case it's a monetary purchase with a non-existing 2011 exchange rate, then we should let `convertTo2011Euro` report the error.